pyproject.toml

```diff
@@ -52,7 +52,11 @@ add-ignore = [
 ]
 
 [tool.pytest.ini_options]
-addopts = "--cov=caf.toolkit --cov-report=xml"
+addopts = '--cov=caf.toolkit --cov-report=xml --doctest-modules -m "not serial"'
+doctest_optionflags = "NORMALIZE_WHITESPACE"
+markers = [
+    "serial: marks tests to not run in parallel (will not run by default)",
+]
```

Comment:
- `--doctest-modules`: turns on running doctests (examples in docstrings within caf.toolkit package)
- `-m "not serial"`: turns off any tests marked with serial
- `doctest_optionflags = "NORMALIZE_WHITESPACE"`: strips whitespace from results and answers when running doctests
- `markers`: registers custom 'serial' marker with pytest so it can be used to mark some (only 1 atm) tests to not run by default

Reply:
Moved the "not serial" flag to tox so its clearer than some tests are ran on a single process in tox, commit 2047fea88df05bfd05befc74bc73278e68b170c3